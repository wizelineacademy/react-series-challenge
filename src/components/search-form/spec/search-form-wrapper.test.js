import React from 'react';
import renderer from 'react-test-renderer';

import { SearchFormWrapper } from '../';

describe('<SearchFormWrapper />', () => {
  test('renders data correctly', () => {
    const tree = renderer.create(<SearchFormWrapper />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
