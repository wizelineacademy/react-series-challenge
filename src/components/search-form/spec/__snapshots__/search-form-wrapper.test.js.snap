// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchFormWrapper /> renders data correctly 1`] = `
<form
  className="sc-htpNat invigS"
/>
`;
